docker/README.md

```diff
@@ -172,3 +171,32 @@ A convenient way to generate a desired number of random bytes and base64-encode
 ```
 $ openssl rand -base64 <desired length>
 ```
+
+## Metrics Exporter
+
+See [monitoring/exporter/README](../monitoring/exporter/README.md) for information on the Exporter.
+Container is also available as a part of the Tinode docker distribution: `tinode/exporter`.
+Run it with
+
+```
+$ docker run -p 6222:6222 -d --name tinode-exporter --network tinode-net \
+		--env SERVE_FOR=<prometheus|influxdb> \
+		--env TINODE_ADDR=<tinode metrics endpoint> \
+		--env LISTEN_AT=":6222" \
+		... <monitoring service specific vars> \
+		tinode/exporter:latest
+```
+
+Available variables:
+| Variable | Type | Default | Function |
+| --- | --- | --- | --- |
+| `SERVE_FOR` | string | `` | Monitoring service: `prometheus` or `influxdb` |
+| `TINODE_ADDR` | string | `http://localhost/stats/expvar/` | Tinode metrics path |
+| `LISTEN_AT` | string | `:6222` | Exporter web server host and port |
+| `INFLUXDB_VERSION` | string | `1.7` | InfluxDB version (`1.7` or `2.0`) |
+| `INFLUXDB_ORGANIZATION` | string | `org` | InfluxDB organization |
+| `INFLUXDB_PUSH_INTERVAL` | int | `60` | Exporter metrics push interval in seconds |
```

Comment:
It just crossed my mind that we need some protection from very short push intervals, i.e. we should not it allow to be under 10s. I'll send a PR.